airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/utils/json_schema_helper.py

```diff
@@ -240,3 +240,17 @@ def _scan_schema(subschema, path=""):
 
     _scan_schema(schema)
     return paths
+
+
+def flatten_tuples(to_flatten):
```

Comment:
This function was orignally imported from the old jsonschema version, it was a private function that disappeared in the newest jsonschema package version, I redeclared it, as it,  here.